package.json

```diff
@@ -28,6 +31,16 @@
     "inquirer": "^9.1.5"
   },
   "devDependencies": {
-    "prettier": "^2.8.7"
+    "eslint": "^8.37.0",
```

Comment:
You also need to add `eslint-config-standard`, otherwise, _ESLint_ will not work correctly.

Reply:
Noted! on it...

Reply:
Changes made! check the recent commit.